.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,23 @@
+<!--
+
+Template: FEATURE REQUEST
+
+This is an issue for a feature request.
+You should label `feature-request`.
+
+
+If your purpose is different, please visit following links.
```

Comment:
Suggestion: "If you issue is not a feature request, please visit one of the following links."

Reply:
👍